### ISSUE TITLE
VCR retry infinte loop

### ISSUE THREAD

**Issue by gerardsn** [nuts-foundation/nuts-node#1423]
Not sure what the exact issue is, but it appears that the VCR rejects invalid VCs received thought the network, but keeps requesting them / trying to proces them.

from logs on stable network after syncing a new node:
```
time="2022-09-08T10:16:42Z" level=error msg="Retry failed" error="verifiable credential must list exactly 2 types" eventSubscriber=vcr_vcs module=Network txRef=4f10640aa1b376a4d29cc8592e9456cb4484383a9fae15bfb51d0179418948a6
time="2022-09-08T10:16:42Z" level=error msg="Retry failed" error="verifiable credential must list exactly 2 types" eventSubscriber=vcr_vcs module=Network txRef=9ca2a588327ff703f959c83e9657496a0d252dd82d6052a77687d11f6cb417c5
time="2022-09-08T10:16:42Z" level=error msg="Retry failed" error="verifiable credential must list exactly 2 types" eventSubscriber=vcr_vcs module=Network txRef=e18c58d784bce1209ca4be006e98ffde913f0113682ab300867b9cbec9eaeda2
...
time="2022-09-08T10:25:13Z" level=error msg="Retry failed" error="verifiable credential must list exactly 2 types" eventSubscriber=vcr_vcs module=Network txRef=4f10640aa1b376a4d29cc8592e9456cb4484383a9fae15bfb51d0179418948a6
time="2022-09-08T10:25:13Z" level=error msg="Retry failed" error="verifiable credential must list exactly 2 types" eventSubscriber=vcr_vcs module=Network txRef=9ca2a588327ff703f959c83e9657496a0d252dd82d6052a77687d11f6cb417c5
time="2022-09-08T10:25:13Z" level=error msg="Retry failed" error="verifiable credential must list exactly 2 types" eventSubscriber=vcr_vcs module=Network txRef=e18c58d784bce1209ca4be006e98ffde913f0113682ab300867b9cbec9eaeda2
```

**Comment by woutslakhorst** [comment]
It has a backoff and max retry count, thus not infinite.....

The question is how it bypassed the publishing node's validation? You can fetch the failed event via the API and get the raw transaction contents.

**Comment by stevenvegt** [comment]
So, perhaps improve log message so an admin knows this is not an infinite loop, but by design.

**Comment by gerardsn** [comment]
> So, perhaps improve log message so an admin knows this is not an infinite loop, but by design.

And/or add it to the frequently encountered errors section of the documentation.

**Comment by gerardsn** [comment]
The events indeed fail after 10 retries. 
```
GET /internal/network/v1/events

[
    {
        "events": [
            {
                "error": "verifiable credential must list exactly 2 types",
                "hash": "4f10640aa1b376a4d29cc8592e9456cb4484383a9fae15bfb51d0179418948a6",
                "retries": 10,
                "transaction": "4f10640aa1b376a4d29cc8592e9456cb4484383a9fae15bfb51d0179418948a6",
                "type": "payload"
            },
            {
                "error": "verifiable credential must list exactly 2 types",
                "hash": "9ca2a588327ff703f959c83e9657496a0d252dd82d6052a77687d11f6cb417c5",
                "retries": 10,
                "transaction": "9ca2a588327ff703f959c83e9657496a0d252dd82d6052a77687d11f6cb417c5",
                "type": "payload"
            },
            {
                "error": "verifiable credential must list exactly 2 types",
                "hash": "e18c58d784bce1209ca4be006e98ffde913f0113682ab300867b9cbec9eaeda2",
                "retries": 10,
                "transaction": "e18c58d784bce1209ca4be006e98ffde913f0113682ab300867b9cbec9eaeda2",
                "type": "payload"
            }
        ],
        "name": "vcr_vcs"
    },
   ...
]

```

- `hash` and `transaction` contain the same value in an event, is this correct?
- What situation is this retry mechanism intended for? The retry window is only long enough to cover temporary hiccups like dropped connections or node/db restarts. (This should be added to the documentation)

**Comment by woutslakhorst** [comment]
I think the count will go up to a 100..... so the window will be longer......

**Comment by reinkrul** [comment]
Issue partially fixed by #1449
